vizro-core/src/vizro/_vizro.py

```diff
@@ -28,9 +27,6 @@ def __init__(self, **kwargs):
         self.dash.config.external_stylesheets.extend(
```

Comment:
@antonymilne - this is important to understand: Our current set-up of specifying external_stylesheets

If a user would like to use another bootstrap theme, this would not work:
`Vizro(external_stylesheets=[dbc.themes.BOOTSTRAP]).build(dashboard).run()`

The current code in `_vizro.py`  always extends to that list, so our `vizro-bootstrap.css` would be read in after the `dbc.themes.Bootstrap`, so the user-defined bootstrap theme has no effect. At this point in time, it's definitely our desired result, as otherwise everything looks completely off.

However, just know, that in case someone still wants to overwrite it, it has to be done like this (right now):

```
  app = Vizro().build(dashboard)
  app.dash.config.external_stylesheets.extend([dbc.themes.BOOTSTRAP])
  app.run()
```

Reply:
Yeah, this is indeed right for now I think. Probably we'll need to change the logic here a bit if and when we introduce CDN as an option anyway.